src/cascadia/TerminalSettingsModel/AIConfig.idl

```diff
@@ -0,0 +1,28 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+#include "IInheritable.idl.h"
+
+namespace Microsoft.Terminal.Settings.Model
+{
+    enum LLMProvider
+    {
+        AzureOpenAI,

```

Comment:
you need a `None` option to return in case none are set - otherwise, the zero value is `AzureOpenAI`

Reply:
Yeah I realized this in #18095 and added it there, I'll bring that change forward to here